benefits/core/templates/core/page.html

```diff
@@ -14,57 +14,50 @@
 
 {% block main_content %}
   <div class="container-fluid">
-    <div class="row main-row">
-      {% if page.noimage %}
-        <div class="container">
-        {% else %}
-          {% load static %}
-          <div class="col-lg-6 image"></div>
-          <div class="col-md-8 offset-md-2 col-lg-6 offset-lg-0">
-          {% endif %}
-          <div class="container content">
-            {% block container_content %}
-
-              {% block icon_title %}
-                {% if page.content_title %}
-                  {% if page.icon %}
-                    {% include "core/includes/icon-title.html" with title=page.content_title icon=page.icon %}
-                  {% else %}
-                    <h1>{{ page.content_title }}</h1>
-                  {% endif %}
-                {% endif %}
-              {% endblock icon_title %}
-
-              {% block paragraph_content %}
-                {% for p in page.paragraphs %}<p>{{ p }}</p>{% endfor %}
-              {% endblock paragraph_content %}
-
-              {% block forms %}
-                {% if page.forms %}
-                  {% for f in page.forms %}
-                    {% include "core/includes/form.html" with form=f %}
+    <div class="container row main-row">
+      {% load static %}
```

Comment:
```suggestion
```
Let's remove this line. I tested the app without it, and it works exactly the same.

_More context_: I found that this is a remnant from when we loaded the image using Django's [`static` tag](https://docs.djangoproject.com/en/4.1/howto/static-files/#configuring-static-files). Notice that in `page.html` at [81fedcf](https://github.com/cal-itp/benefits/commit/81fedcfca204be24df17fcfd23018f6ed5fa1b80#diff-c1474fc80660fe6368a980de34390440c3f9e4126d49bcaa713613abe8e25cb2R14-R16), we loaded the image url using `{% static page.image.src %}`. [This commit](https://github.com/cal-itp/benefits/commit/7956b5873ea1791aa2b74384f50ebcf53dbdb91f#diff-3b4760037985a2db7dd1b70354c449c5951061b602d1285f40a274de10719355L18) removed usage of `static` so need to load it anymore.

Reply:
Done in https://github.com/cal-itp/benefits/pull/932/commits/96aa1dc42e1546975eea5adc59494ad19b6938b0